sensor_msgs/include/sensor_msgs/impl/point_cloud2_iterator.hpp

```diff
@@ -159,23 +159,6 @@ inline void PointCloud2Modifier::clear()
 }
 
 
-/**
- * @brief Function setting some fields in a PointCloud and adjusting the
- *        internals of the PointCloud2
- * @param n_fields the number of fields to add. The fields are given as
- *        triplets: name of the field as char*, number of elements in the
- *        field, the datatype of the elements in the field
- *
- * E.g, you create your PointCloud2 message with XYZ/RGB as follows:
- * <PRE>
- *   setPointCloud2FieldsByString(cloud_msg, 4, "x", 1, sensor_msgs::msg::PointField::FLOAT32,
- *                                              "y", 1, sensor_msgs::msg::PointField::FLOAT32,
- *                                              "z", 1, sensor_msgs::msg::PointField::FLOAT32,
- *                                              "rgb", 1, sensor_msgs::msg::PointField::FLOAT32);
- * </PRE>
- * WARNING: THIS DOES NOT TAKE INTO ACCOUNT ANY PADDING AS DONE UNTIL HYDRO
- * For simple usual cases, the overloaded setPointCloud2FieldsByString is what you want.
- */
```

Comment:
Is this documentation block removed because it's an implementation file?

Reply:
Yes, exactly.  Doxygen was complaining that `n_fields` was documented twice, and that was because we had the same docblock on the declaration and definition.  Removing one of them gets rid of that warning.